src/components/VenueFilter/VenueFilter.test.js

```diff
@@ -0,0 +1,28 @@
+import { render, renderHook, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import VenueFilter from ".";
+import { useFilterStore } from "../../store";
+
+test("renders one tag input field and one clear button", () => {
+  render(<VenueFilter />);
+  const tagsInput = screen.getByRole("searchbox", { placeholder: /search/i });
+  const clearButton = screen.getByRole("button", { name: /clear/i });
+  expect(tagsInput).toBeInTheDocument();
+  expect(clearButton).toBeInTheDocument();
+});
+
+test("clicking the clear button resets active tags", async () => {
+  const store = renderHook(() => useFilterStore());
+  const { venueKeywords } = store.result.current;
+  const user = userEvent.setup();
+
+  render(<VenueFilter />);
+  const clearButton = screen.getByRole("button", { name: /clear/i });
+
+  await user.click(clearButton);
```

Comment:
I would expect to test the length before clicking the clear button to make sure that the button works as expected.

Reply:
Makes sense! Thank you 🙏